thrift/admin.thrift

```diff
@@ -301,4 +328,32 @@ struct ResendReplicationTasksRequest {
   60: optional i64 (js.type = "Long") startVersion
   70: optional i64 (js.type = "Long") endEventID
   80: optional i64 (js.type = "Long") endVersion
-}
\ No newline at end of file
+}
+
+struct GetDynamicConfigRequest {
+  10: optional string configName
+  20: optional list<configStore.DynamicConfigFilter> filters
+}
+
+struct GetDynamicConfigResponse {
+  10: optional shared.DataBlob value
```

Comment:
if filter is not specified, all values for the requested configName will be returned as a datablob?